ts/features/wallet/onboarding/privative/screens/add/AddPrivativeCardScreen.tsx

```diff
@@ -1,24 +1,111 @@
+import { fromNullable } from "fp-ts/lib/Option";
 import { View } from "native-base";
 import * as React from "react";
+import { useEffect, useState } from "react";
+import { SafeAreaView } from "react-native";
 import { connect } from "react-redux";
 import { Dispatch } from "redux";
-import { GlobalState } from "../../../../../../store/reducers/types";
+import { PaymentInstrument } from "../../../../../../../definitions/pagopa/walletv2/PaymentInstrument";
+import { Body } from "../../../../../../components/core/typography/Body";
+import { H1 } from "../../../../../../components/core/typography/H1";
+import { H3 } from "../../../../../../components/core/typography/H3";
+import { IOStyles } from "../../../../../../components/core/variables/IOStyles";
 import BaseScreenComponent from "../../../../../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../../../../../components/ui/FooterWithButtons";
+import I18n from "../../../../../../i18n";
+import { GlobalState } from "../../../../../../store/reducers/types";
+import { emptyContextualHelp } from "../../../../../../utils/emptyContextualHelp";
+import {
+  cancelButtonProps,
+  confirmButtonProps
+} from "../../../../../bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+import {
+  fold,
+  getValueOrElse
+} from "../../../../../bonus/bpd/model/RemoteValue";
+import {
+  addPrivativeToWallet,
+  walletAddPrivativeCancel,
+  walletAddPrivativeCompleted
+} from "../../store/actions";
+import { onboardingPrivativeAddingResultSelector } from "../../store/reducers/addingPrivative";
+import { onboardingPrivativeFoundSelector } from "../../store/reducers/foundPrivative";
 
 type Props = ReturnType<typeof mapDispatchToProps> &
   ReturnType<typeof mapStateToProps> &
-  Pick<React.ComponentProps<typeof BaseScreenComponent>, "contextualHelp">;;
+  Pick<React.ComponentProps<typeof BaseScreenComponent>, "contextualHelp">;
+
+const AddPrivativeBody = (props: Props) => (
+  <>
+    <H1>TMP Privative card display screen</H1>
+    <Body>
+      Card Number:
+      {props.foundPrivative.fold("No card number", pm => pm.panPartialNumber)}
+    </Body>
+  </>
+);
 
 /**
  * This screen shows the user the private card found and allows him to add it to the wallet
  * @param props
  * @constructor
  */
-const AddPrivativeCardScreen = (_: Props): React.ReactElement => <View />;
+const AddPrivativeCardScreen = (props: Props): React.ReactElement => {
+  const [stateLabel, setStateLabel] = useState("undefined");
+  useEffect(() => {
+    fold(
+      props.addingResult,
+      () => setStateLabel(""),
```

Comment:
since the initial state is "undefined" 
does make it sense to place also here?
```suggestion
      () => setStateLabel("undefined"),
```

Reply:
Was undefined in the previous version, but I've removed because is not an information in the draft `Add screen` for the user.